src/main/java/htsjdk/tribble/util/ParsingUtils.java

```diff
@@ -80,9 +83,7 @@
 
     public static InputStream openInputStream(String path)
             throws IOException {
-
-        InputStream inputStream;
-
+        final InputStream inputStream;
         if (path.startsWith("http:") || path.startsWith("https:") || path.startsWith("ftp:")) {
             inputStream = getURLHelper(new URL(path)).openInputStream();
         } else if (IOUtil.hasScheme(path)) {
```

Comment:
Should `SeekablePathStream` be used here as well, or does it have to be `Files.newInputStream()` for some reason?